scripts/docs/build.sh

```diff
@@ -3,7 +3,7 @@ set -eux
 
 # DEV: unless it's built with editable, following sphinx-build fails
 if [ -z ${CIRCLECI+x} ]; then
-  CMAKE_BUILD_PARALLEL_LEVEL=$(nproc) pip install -v -e .
+  CMAKE_BUILD_PARALLEL_LEVEL=$(nproc) CARGO_BUILD_JOBS=$(nprocs) pip install -v -e .
```

Comment:
nit: What's the difference between `$(nprocs)` and `$(nproc)` here? Do they not return the same value?

Reply:
`nproc` exists, and `nprocs` doesn't. good catch!

```suggestion
  CMAKE_BUILD_PARALLEL_LEVEL=$(nproc) CARGO_BUILD_JOBS=$(nproc) pip install -v -e .
```